### DISCUSSION TITLE
What is the National game of 🇮🇳?

### DISCUSSION THREAD

**Discussion by Sajansharma0017** [ParanoidUser/yolo#1205] [Galaxy Trivia] [answered]
What is the National game of INDIA?

**Comment by tomerf** [comment, marked as answer]
The Indian government has not designated any official sport as India's national sport. Because of its widespread popularity, hockey is widely regarded as India's national sport. However, India does not have an official national game, and hockey and Kabaddi are frequently mistaken for India's national sports.

**Comment by Sajansharma0017** [comment]
Absolutely Right!
